api/src/main/java/org/openmrs/util/OpenmrsConstants.java

```diff
@@ -705,8 +712,11 @@ public static final List<GlobalProperty> CORE_GLOBAL_PROPERTIES() {
 		
 		props.add(new GlobalProperty(GP_MAIL_SMTP_STARTTLS_ENABLE, "false",
 		        "Set to true to enable TLS encryption, else set to false"));
-		
+
 		props.add(new GlobalProperty(GP_HOST_URL, "",
```

Comment:
Since we deprecated it, let us remove it from here such that we do not create it in new installations.

Reply:
@dkayiwa I removed it in the new commit.